view/frontend/web/js/view/payment/cart/checkoutcom_applepay_cart.js

```diff
@@ -372,7 +372,32 @@ require([
          */
         function getSupportedNetworks()
         {
-            return getValue("supported_networks").split(",");
+            let networksEnabled = getValue("supported_networks").split(",");
+            return processSupportedNetworks(networksEnabled);
+        }
+
+        /**
+         * Process supported network based on store country (SA)
+         *
+         * @return {array}
+         */
+        function processSupportedNetworks(networksEnabled)
+        {
+            if (networksEnabled.includes("mada")) {
+                return Utilities.getStoreCountry == "SA" ? networksEnabled : networksEnabled.splice(networksEnabled.indexof("mada"), 1);
```

Comment:
@ved-beejadhur-cko Just want to check something because I don't have all the details

What should happen if a merchant is in the UK but wants to accept mada payments? Is the possible?